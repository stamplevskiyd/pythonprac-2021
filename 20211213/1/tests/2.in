[31, 67, 44, 88, 21, 2, 46, 87, 60, 37, 1, 42, 64, 39, 67, 100]
