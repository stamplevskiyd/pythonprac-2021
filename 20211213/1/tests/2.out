[1, 2, 21, 31, 37, 39, 42, 44, 46, 60, 64, 67, 67, 87, 88, 100]
